src/utils/container.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Checks if the list includes any value from the search.
+ * @param list The list to search in.
+ * @param search The values to search.
+ * @returns {boolean} _true_ if the list includes any value from the search.
+ */
+export function includesAny(list: string | string[] | null | undefined, search: string | string[]) {
+    if (!list) {
+        return true;
+    }
+
+    let negative = false;
+
+    if (typeof list === 'string') {
+        if (list.startsWith('-')) {
+            negative = true;
+            list = list.substring(1);
+        }
+
+        list = list.split(',');
+    }
+
+    list = list.filter(i => i);
+
+    if (!list.length) {
+        return true;
+    }
+
+    if (typeof search === 'string') {
+        search = search.split(',');
+    }
+
+    search = search.filter(i => i);
+
+    const holdList = list;
```

Comment:
Why reassign this?

Reply:
Lint error in the callback function. Something like `'list' can be 'null'`.

Reply:
Hmm, it is not Lint - it is VS Code and TS checker (?) during build: `TS2533: Object is possibly 'null' or 'undefined'.`

Reply:
I used non-null assertion operator and disabled eslint rule.